.github/workflows/smoke-test.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         git submodule update --init
         pip install -e .[dev]
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        layup bootstrap
```

Comment:
We shouldn't need to add this - we should have layup automatically install these files if they're not already installed when it first trys to run these commands.

Reply:
Maybe this should be a ticket @drewoldag ?